parser/parser.go

```diff
@@ -110,3 +122,25 @@ func (p *parser) many(start lexer.Type, end lexer.Type, cb func()) {
 	}
 	p.next()
 }
+
+func (p *parser) some(start lexer.Type, end lexer.Type, cb func()) {
+	p.mark()
+
+	hasDef := p.skip(start)
+	if !hasDef {
+		return
+	}
+
+	called := false
+	for p.peek().Kind != end && p.err == nil {
+		called = true
+		cb()
+	}
+
+	if !called {
```

Comment:
should we give some error here?
however, we do not know what cb is requesting.

error by https://www.npmjs.com/package/graphql
`GraphQLError: Syntax Error: Expected Name, found }`